### PR TITLE
add disk setting into suggestionConfiguration

### PR DESCRIPTION
**Pull request by sperlingxx**
The aim of this PR is appending the setting of disk(EphemeralStorage) into configuration of suggestion service.

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/katib/989)

### CODE REVIEW

**Comment by gaocegege** [comment]
@sperlingxx 
Thanks for your contribution! :tada: :+1:

Do you meet any problem with the disk when you use katib? Did these suggestions consume much disk space?

File: pkg/util/v1alpha3/katibconfig/config.go [hunk 1]
```diff
@@ -42,31 +42,41 @@ func GetSuggestionConfigData(algorithmName string, client client.Client) (map[st
...
 			memLimit := suggestionConfig.Resource.Limits[corev1.ResourceMemory]
+			diskLimit := suggestionConfig.Resource.Limits[corev1.ResourceEphemeralStorage]
 			if !cpuLimit.IsZero() {
 				suggestionConfigData[consts.LabelSuggestionCPULimitTag] = cpuLimit.String()
 			}
 			if !memLimit.IsZero() {
 				suggestionConfigData[consts.LabelSuggestionMemLimitTag] = memLimit.String()
 			}
+			if !diskLimit.IsZero() {
+				suggestionConfigData[consts.LabelSuggestionDiskRequestTag] = diskLimit.String()
```
Comment (gaocegege) [inline pkg/util/v1alpha3/katibconfig/config.go]:
```suggestion
				suggestionConfigData[consts.LabelSuggestionDiskLimitTag] = diskLimit.String()
```

**Comment by sperlingxx** [comment]
> @sperlingxx
> Thanks for your contribution! 🎉 👍
> 
> Do you meet any problem with the disk when you use katib? Did these suggestions consume much disk space?

Thanks for so quick reviewing.  I add this setting for two concerns: 
-  I want to minimize the cost of disk when I debug Katib with local machine(Minikube). 
-  All resource limits/requests must be explicitly defined for creating pods on customized k8s cluster of Ant Finanical.

**Comment by gaocegege** [comment]
SGTM. Thanks.

https://github.com/kubeflow/katib/pull/989#discussion_r361914764 One typo-fix suggestion here.

**Comment by gaocegege** [comment]
Thanks for your contribution! :tada: :+1:

**Pull request merged**